src/hostnet/lib/slirp.ml

```diff
@@ -124,17 +124,21 @@ let connect x peer_ip local_ip =
                       end else begin
                         let payload = Cstruct.sub udp Wire_structs.sizeof_udp (length - Wire_structs.sizeof_udp) in
                         let for_us = Ipaddr.V4.compare dst local_ip == 0 in
+                        let for_extra_dns = Ipaddr.V4.compare dst extra_dns_ip == 0 in
```

Comment:
these `==` should just be `=`. 

`=` is structural equality and `==` is for physical (pointer) equality in OCaml.

Doesnt matter too much for integers (which are immediates) but better for code style to uniformly use structural equality.